python/ml4ir/base/config/keys.py

```diff
@@ -21,6 +21,7 @@ class ArchitectureKey(Key):
     DNN = "dnn"
     LINEAR = "linear"
     RNN = "rnn"
+    DENSE_MODEL = "dense-model"
```

Comment:
@arvindsrikantan Let's rename this. `dense-model` is quite meaningless to what you are really doing. Maybe `dnn-dag` or `dense-dag`?

Reply:
I still like `hierarchical-dnn` too.